src/wasm-lib/kcl/src/parser/parser_impl.rs

```diff
@@ -949,25 +949,31 @@ fn if_expr(i: TokenSlice) -> PResult<BoxNode<IfExpression>> {
 }
 
 // Looks like
-// (arg0, arg1) => {
+// (arg0, arg1) {
 //     const x = arg0 + arg1;
 //     return x
 // }
```

Comment:
Am I understanding it right that we allow function expressions that look like this, without `fn` before it?

Reply:
I'm pretty sure it works today without it, e.g., in the example here: https://zoo.dev/docs/kcl/reduce

I guess we should require something, or else there will be a back-compat hazard with parsing and `(...) {...}` is a little subtle for a function declaration.

Requiring `fn` seems like a good idea, but let me have a think

Reply:
It works today without fn, but we require =>. I honestly wouldn't worry about breaking code in this case. Anyone using anonymous functions isn't going to have a problem upgrading.

Reply:
Implemented in the latest commit